### PR TITLE
Limit caert qty in checkout page

### PR DESCRIPTION
**Pull request by priyankavasoya**
@harishmk1805 
Please check

### CODE REVIEW

File: app/code/Dss/Limitcartqty/Api/DataConfigInterface.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+namespace Dss\Limitcartqty\Api;
+
+interface DataConfigInterface
+{
+    /**
+     * MinValue
+     *
+     * @return mixed
+     */
+    public function getMinValue();
```
Comment (harishmk1805) [inline app/code/Dss/Limitcartqty/Api/DataConfigInterface.php:28]: return type missing

File: app/code/Dss/Limitcartqty/composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (harishmk1805) [inline app/code/Dss/Limitcartqty/composer.json:1]: remove space

File: app/code/Dss/Limitcartqty/etc/adminhtml/system.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        </tab>
+        <section id="Dss_Commerce" translate="label" type="text" sortOrder="100" showInDefault="1"
+            showInWebsite="1" showInStore="1">
+            <label>LimitCartQty</label>
+            <tab>bss</tab>
+            <resource>Dss_Limitcartqty::config</resource>
+            <group id="item_options" translate="label" type="text" sortOrder="50" showInDefault="1"
+                showInWebsite="1" showInStore="1">
+                <label>Product Stock Options</label>
+                <field id="Dss_min_total_qty" translate="label" sortOrder="20" showInDefault="1"
```
Comment (harishmk1805) [inline app/code/Dss/Limitcartqty/etc/adminhtml/system.xml:29]: use proper name for id and name

File: app/code/Dss/Limitcartqty/etc/module.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ * @category   Dss
+ * @package    Dss_Limitcartqty
+ * @author     Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Dss_Limitcartqty">
+    <sequence>
```
Comment (harishmk1805) [inline app/code/Dss/Limitcartqty/etc/module.xml:18]: please formate this code line

**Review by harishmk1805 — CHANGES_REQUESTED** [review]

File: app/code/Dss/Limitcartqty/Api/DataConfigInterface.php (hunk 1 above)
Comment (priyankavasoya) [reply app/code/Dss/Limitcartqty/Api/DataConfigInterface.php]: @harishmk1805 
Done.

File: app/code/Dss/Limitcartqty/composer.json (hunk 2 above)
Comment (priyankavasoya) [reply app/code/Dss/Limitcartqty/composer.json]: @harishmk1805 
removed.

File: app/code/Dss/Limitcartqty/etc/adminhtml/system.xml (hunk 3 above)
Comment (priyankavasoya) [reply app/code/Dss/Limitcartqty/etc/adminhtml/system.xml]: @harishmk1805 
use proper name for id and name added.

File: app/code/Dss/Limitcartqty/etc/module.xml (hunk 4 above)
Comment (priyankavasoya) [reply app/code/Dss/Limitcartqty/etc/module.xml]: @harishmk1805 
done.

**Review by harishmk1805 — APPROVED** [review]

**Pull request merged**